### PR TITLE
Scorecard updates

### PR DESCRIPTION
**Pull request by eanders**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

WIP

## Type of change
[//]: # 'remove options that are not relevant'
- [x] New feature (adds functionality)

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

**Comment by eanders** [comment]
@dtgreiner there is one outstanding question on the spec, but it should be a one line fix.  I believe this is ready for review at this point.

File: drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_edit_js.haml [hunk 1]
```diff
@@ -13,4 +13,23 @@
...
+      // Prevent both 6a and 6b from being selected (per spec)
+      var no_concern = '.report-form input[name="boston_project_scorecard_report[no_concern]"]'
+      var no_concern_hidden = '.report-form input[name="boston_project_scorecard_report[no_concern]"][type="hidden"]'
+      var materials_concern = '.report-form input[name="boston_project_scorecard_report[materials_concern]"]'
+      var materials_concern_hidden = '.report-form input[name="boston_project_scorecard_report[materials_concern]"][type="hidden"]'
+      $(no_concern + ',' + materials_concern).on('change', function(e) {
+        var clicked_value = e.currentTarget.value;
+        // -1 is Not Applicable, that's allowed to be chosen in both situations, so do nothing in that case
+        if(clicked_value != -1) {
+          console.log($(e.currentTarget).attr('name'))
```
Comment (dtgreiner) [inline drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_edit_js.haml]:
```suggestion
```

File: drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_policy_alignment.haml
Comment (dtgreiner) [inline drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_policy_alignment.haml:1]: Looks like this file needs Q19/Q20 Racial Equity questions to be bumped up to Q21/Q22

File: drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_financial_performance.haml [hunk 2]
```diff
@@ -42,11 +42,13 @@
...
+        12. Is the project efficient with funds? Is the cost per person served in the project (using the total grant award) aligned with the average cost per person across the CoC portfolio? ($19,497 per person for PSH and $14,022 per person for RRH)
       %td
-        Project Application Budget
-        %p
-        Most Recently Completed Grant Year Spend divided by Total Number of Persons Served
+        Project APR
+        %br
+        Q07a: Number of Persons Served
+        %br
+        Q28 Financial Information: Total Expenditures
```
Comment (dtgreiner) [inline drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_financial_performance.haml:51]: Should these be linking to the APR similar to the other questions?

**Review by dtgreiner — COMMENTED** [review]
@eanders Just a couple small things here, but looks good other than that.

File: drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_financial_performance.haml (hunk 2 above)
Comment (eanders) [reply drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_financial_performance.haml:51]: Q28 isn't part of the APR that gets run from within HMIS, I believe it is hand-entered, so no link here.

File: drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_policy_alignment.haml
Comment (eanders) [reply drivers/boston_project_scorecard/app/views/boston_project_scorecard/warehouse_reports/scorecards/_policy_alignment.haml:1]: Thanks!

**Pull request merged**